certbot-apache/certbot_apache/_internal/configurator.py

```diff
@@ -51,6 +50,43 @@
 logger = logging.getLogger(__name__)
 
 
+class _OsOptions:
+    def __init__(self,
+                 server_root="/etc/apache2",
+                 vhost_root="/etc/apache2/sites-available",
+                 vhost_files="*",
+                 logs_root="/var/log/apache2",
+                 ctl="apache2ctl",
+                 version_cmd: Optional[List[str]] = None,
+                 restart_cmd: Optional[List[str]] = None,
+                 restart_cmd_alt: Optional[List[str]] = None,
+                 conftest_cmd: Optional[List[str]] = None,
+                 enmod: Optional[str] = None,
+                 dismod: Optional[str] = None,
+                 le_vhost_ext="-le-ssl.conf",
+                 handle_modules=False,
+                 handle_sites=False,
+                 challenge_location="/etc/apache2",
+                 apache_bin: Optional[str] = None,
```

Comment:
nit: Any reason not to just use `bin` here? I think all other attributes have the same name. Why make this one different?

Reply:
Yes, my IDE and pylint are complaining about `bin` shadowing the built-in `bin()` function. So I changed the naming of this parameter in the constructor to fix that.